server/src/main/resources/reference.conf

```diff
@@ -11,3 +11,5 @@ staging-bucket-name = ${STAGING_BUCKET}
 clean-up-lock-interval = 7 minutes
 flush-cache-interval = 10 minutes
 sync-cloud-storage-directory-interval = 15 minutes
+delocalize-directory-interval = 15 seconds
```

Comment:
maybe every 60 seconds? every 15 seconds feels really frequent 🤔

Reply:
hmm I did every 15 as a suggestion from @rtitle but am open to discussion on how frequently we want to delocalize